app/models/jurisdiction.rb

```diff
@@ -3,6 +3,8 @@ class Jurisdiction < ActiveRecord::Base
   has_many :office_jurisdictions
   has_many :offices, through: :office_jurisdictions
 
+  has_many :business_entities
```

Comment:
Do we *need* this here and on office?

The join is only used in tests, not in the code.